vector/src/main/java/im/vector/app/features/html/EventHtmlRenderer.kt

```diff
@@ -40,15 +48,35 @@ import javax.inject.Singleton
 class EventHtmlRenderer @Inject constructor(
         htmlConfigure: MatrixHtmlPluginConfigure,
         context: Context,
-        vectorPreferences: VectorPreferences
+        vectorPreferences: VectorPreferences,
+        private val activeSessionHolder: ActiveSessionHolder
 ) {
 
     interface PostProcessor {
         fun afterRender(renderedText: Spannable)
     }
 
     private val builder = Markwon.builder(context)
-            .usePlugin(HtmlPlugin.create(htmlConfigure))
+            .usePlugins(listOf(
```

Comment:
Nit: maybe use several call to `usePlugin` for readability, instead of using `usePlugins`, but not a blocker.